source/backend/tests/unit/api/Services/ProjectServiceTest.cs

```diff
@@ -52,18 +65,13 @@ public void Search_Success()
         public void Search_NoPermission()
         {
             // Arrange
-            var helper = new TestHelper();
-            var user = PrincipalHelper.CreateForPermission();
-            var service = helper.Create<ProjectService>(user);
-
-            var repository = helper.GetService<Mock<IProjectRepository>>();
+            var service = CreateProjectServiceWithPermissions();
 
             // Act
             Action act = () => service.SearchProjects("some string", 1);
 
             // Assert
             act.Should().Throw<NotAuthorizedException>();
-            repository.Verify(x => x.SearchProjects(It.IsAny<string>(), It.IsAny<int>()), Times.Never);
```

Comment:
There are a few tests below that would need to follow new pattern. They are still asserting **Times.Never** on repo usage when exceptions are thrown

Reply:
Right, these likely came from the merge. Will update.